src/operator/quantization/quantized_batch_norm.cc

```diff
@@ -135,11 +135,9 @@ NNVM_REGISTER_OP(BatchNorm)
     }
     return node;
   })
-.set_attr<FAvoidQuantizeInput>("FAvoidQuantizeInput", [](const NodeAttrs &attrs, size_t index) {
-  if (index == 0)
-    return false;
-  else
-    return true;
+.set_attr<FAvoidQuantizeInput>("FAvoidQuantizeInput", [](
+  const NodeAttrs &attrs, const size_t index, const std::string quantize_granularity) {
+  return (index == 0) ? false : true;
```

Comment:
`return (index != 0);` ?

Reply:
No, the attribute of this op is different than others. Check [here](https://github.com/apache/incubator-mxnet/blob/master/src/operator/quantization/quantized_batch_norm.cc#L139).

Reply:
good.